protocol/communities_messenger_test.go

```diff
@@ -63,9 +108,9 @@ func (s *MessengerCommunitiesSuite) SetupTest() {
 	s.shh = gethbridge.NewGethWakuWrapper(shh)
 	s.Require().NoError(shh.Start())
 
-	s.admin = s.newMessenger()
-	s.bob = s.newMessenger()
-	s.alice = s.newMessenger()
+	s.admin = s.newMessenger(AdminPassword)
+	s.bob = s.newMessenger(BobPassword)
+	s.alice = s.newMessenger(AlicePassword)
```

Comment:
For the tests do we need to define the password at this level? Could passwords be an empty string or the same password used all messengers? 

Function signatures have been changed to accommodate the concept of password, but the password is not the testable element.

Reply:
The problem is that join to community requires password, and if its not match the behaviour should be different.  An i suppose we should test that. If i set same password for each community member, test may not cover some edge case.